wrapping/itkCuberilleImageToMeshFilter.wrap

```diff
@@ -2,8 +2,9 @@ itk_wrap_class("itk::CuberilleImageToMeshFilter" POINTER_WITH_2_SUPERCLASSES)
 UNIQUE(types "${WRAP_ITK_SCALAR}")
 foreach(d ${ITK_WRAP_IMAGE_DIMS})
   foreach(t ${types})
-    itk_wrap_template("${ITKM_I${t}${d}}M${ITKM_${t}}${d}"
-                      "${ITKT_I${t}${d}}, itk::Mesh< ${ITKT_${t}},${d} >, itk::InterpolateImageFunction< ${ITKT_I${t}${d}}, ${ITKT_D} >")
+    # in all cases use float for mesh coordinates and double for interpolator's internal coordinates
+    itk_wrap_template("${ITKM_I${t}${d}}M${ITKM_F${d}}"
```

Comment:
The first template paramater is `TPixelType` instead of the coordinate type. I think we want the same pixel type as the input image pixel type for most cases.

If a float pixel type is needed, it could be wrapped in addition.

Reply:
You are right.